modules/govuk/manifests/apps/content_data_api/rabbitmq.pp

```diff
@@ -30,16 +30,33 @@
   $amqp_user  = 'content_data_api',
   $amqp_pass = undef,
   $amqp_exchange = 'published_documents',
+  $amqp_dlx = 'content_data_api_dlx',
   $amqp_queue = 'content_data_api',
-  $amqp_bulk_importing_queue = 'content_data_api_govuk_importer'
+  $amqp_bulk_importing_queue = 'content_data_api_govuk_importer',
+  $amqp_dead_letter_queue = 'content_data_api_dead_letter_queue'
```

Comment:
Ideally these new parameters would also be documented in the comment above. Especially the `amqp_dlx` on, as that's lots of letters...